### PR TITLE
[IOS-545] Hot fix - onboard fix

### PR DESCRIPTION
**Pull request by bwdowikowski**
### Ticket
[IOS-545](https://netguru.atlassian.net/browse/IOS-545)

### Task Description
Added skip button in 4th step in onboarding
Changed image in 4th step in obnoarding

### Aditional Notes (optional)
I'm not sure if protocol for skipping step in onboarding would better call than current state of method in `ShotsCollectionViewController.skipStep()`?

### CODE REVIEW

File: Inbbbox/Resources/pt-PT.lproj/Localizable.strings [hunk 1]
```diff
@@ -335,6 +335,7 @@
 "ShotsOnboardingStateHandler.Onboarding-Step3" = "onboarding-step3-pt";
 "ShotsOnboardingStateHandler.Onboarding-Step4" = "onboarding-step4-pt";
 "ShotsOnboardingStateHandler.Onboarding-Step5" = "onboarding-step5-pt";
+"ShotsOnboardingStateHandler.Skip" = "$Skip this step";
```
Comment (pikor) [inline Inbbbox/Resources/pt-PT.lproj/Localizable.strings]: Let's deliver this hot fix without translations (so please remove `$`) and add this text into our document for missing translations.

File: Inbbbox/Source Files/Protocols/OnboardingSkipButtonHandlerDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+//
+//  OnboardingSkipable.swift
+//  Inbbbox
+//
+//  Created by Blazej Wdowikowski on 11/25/16.
+//  Copyright © 2016 Netguru Sp. z o.o. All rights reserved.
+//
+
+protocol OnboardingSkipButtonHandlerDelegate: class {
+    // Called when skipButton should appear on centrain situations
```
Comment (pikor) [inline Inbbbox/Source Files/Protocols/OnboardingSkipButtonHandlerDelegate.swift]: Typo `certain`

File: Inbbbox/Source Files/Protocols/OnboardingSkipButtonHandlerDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    func shouldSkipButtonAppear()
+    // Called when skipButton should disappear on centrain situations
```
Comment (pikor) [inline Inbbbox/Source Files/Protocols/OnboardingSkipButtonHandlerDelegate.swift]: Same here.

**Comment by netguru-hound** [comment]
## [Current coverage](https://codecov.io/gh/netguru/inbbbox-ios/pull/331?src=pr) is 31.67% (diff: 49.41%)
> Merging [#331](https://codecov.io/gh/netguru/inbbbox-ios/pull/331?src=pr) into [release/1.0.4](https://codecov.io/gh/netguru/inbbbox-ios/branch/release/1.0.4?src=pr) will increase coverage by **0.04%**

```diff
@@           release/1.0.4       #331   diff @@
===============================================
  Files                256        257     +1   
  Lines              11444      11507    +63   
  Methods                0          0          
  Messages               0          0          
  Branches               0          0          
===============================================
+ Hits                3620       3645    +25   
- Misses              7824       7862    +38   
  Partials               0          0          
```

> Powered by [Codecov](https://codecov.io?src=pr). Last update [43e3ecf...a86c822](https://codecov.io/gh/netguru/inbbbox-ios/compare/43e3ecf1e2c4019fb21e40c289e19e10bb20ac4f...a86c822c1b05b361c67498fbfbffa78c850169c4?src=pr)

**Pull request merged**